src/main/java/org/jabref/gui/linkedfile/LinkedFileDialogController.java

```diff
@@ -18,7 +18,10 @@
 import com.airhacks.afterburner.views.ViewLoader;
 import jakarta.inject.Inject;
 
-public class LinkedFileEditDialogView extends BaseDialog<LinkedFile> {
+public class LinkedFileDialogController extends BaseDialog<LinkedFile> {
```

Comment:
This would be an architectural pattern shift. Read about what a model, view and controller is supposed to do, and also about the MVVM pattern followed in JabRef's JavaFX modules:
https://devdocs.jabref.org/code-howtos/javafx.html

@koppor is a controller needed here? the renaming seems unnecessary to me.

Reply:
AFAIK, we always call it "View". We have background material collected at https://devdocs.jabref.org/code-howtos/javafx.html#architecture-model---view---controller---viewmodel-mvcvm.

Please revert the change, afterwards, @calixtus should double check on this.